File: projects/Java/src/examples/PrintJson.java
package examples;

import com.chronoxor.test.*;

public class PrintJson
{
    public static void main(String[] args)
    {
        System.out.println(new StructSimple().toJson());
        System.out.println();

        System.out.println(new StructOptional().toJson());
        System.out.println();

        System.out.println(new StructNested().toJson());
        System.out.println();
    }
}

File: projects/Kotlin/src/examples/PrintJson.kt
package examples

import com.chronoxor.test.*

object PrintJson
{
    @JvmStatic
    fun main(args: Array<String>)
    {
        println(StructSimple().toJson())
        println()

        println(StructOptional().toJson())
        println()

        println(StructNested().toJson())
        println()
    }
}
